engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectModulesScreen.java

```diff
@@ -45,11 +45,7 @@
 import org.terasology.rendering.nui.databinding.Binding;
 import org.terasology.rendering.nui.databinding.ReadOnlyBinding;
 import org.terasology.rendering.nui.itemRendering.AbstractItemRenderer;
-import org.terasology.rendering.nui.widgets.TextChangeEventListener;
-import org.terasology.rendering.nui.widgets.UIButton;
-import org.terasology.rendering.nui.widgets.UILabel;
-import org.terasology.rendering.nui.widgets.UIList;
-import org.terasology.rendering.nui.widgets.UIText;
+import org.terasology.rendering.nui.widgets.*;
```

Comment:
No star imports, please. (Go to _Settings → Editor → Code Style → Java → Imports_ and set _"Class count to use import with '*'"_ along with _"Names count to use static import with '*'"_ to 999)